tests/drivers/spi/spi_loopback/boards/mimx8mm_phyboard_polis_mimx8mm6_m4.overlay

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2024 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_imx/mimx8ml8dvnlz-pinctrl.dtsi>
+
+&pinctrl {
+	ecspi1_default: ecspi1_default {
+		group1 {
+			pinmux = <&iomuxc_ecspi1_miso_ecspi_miso_ecspi1_miso>,
+					 <&iomuxc_ecspi1_mosi_ecspi_mosi_ecspi1_mosi>,
+					 <&iomuxc_ecspi1_sclk_ecspi_sclk_ecspi1_sclk>;
+			slew-rate = "fast";
+			drive-strength = "x6";
+		};
+		group2 {
+			pinmux = <&iomuxc_ecspi1_ss0_ecspi_ss_ecspi1_ss0>;
+			slew-rate = "fast";
+			drive-strength = "x6";
+			bias-pull-up;
+		};
+	};
+};
```

Comment:
per my understand, pinctrl should be board specified,  but have no relation with test case, right? so move pinctrl property of scspi1 to board dts? so that can reused by other application or test case.

Reply:
Yes, this seems better.